packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -286,11 +285,6 @@ export class ConnectionService {
     }
 
     const connection = JsonTransformer.fromJSON(connectionJson, Connection)
-    try {
```

Comment:
Validation is in fromJSON now. Not sure we want to (re)raise the error here. Left out for now

Reply:
You've disabled validation, so we're losing the validation here now

Reply:
yes eventually i disabled it, because it got a little messy with the current tests.